whylog/tests/tests_client/test_files/003_match_time_range/parsers.yaml

```diff
@@ -1,10 +1,12 @@
 ---
 name: cause
-regex: ^(\d+) root cause
-params:
-    - {type: time, format: '%s'}
+regex_str: ^(\d+) root cause
+primary_key_groups: [1]
+log_type: default
+convertions: {1: date}
 ---
+primary_key_groups: [1]
+log_type: default
 name: effect
-regex: ^(\d+) visible effect$
-params:
-    - {type: time, format: '%s'}
+regex_str: ^(\d+) visible effect$
```

Comment:
This parser does not match with line content: "2015-12-03 12:08:09 visible effect"